### PR TITLE
feat(ci): created initial github workflow

### PR DESCRIPTION
**Pull request by Veetaha**
Closes: #3
Created initial ci workflow with Telegram notifications.
Telegram notifications are sent upon each push to any branch.
Created separate steps for lining, building, and testing and a dedicated job for deployment, too (this may be converted just into a step instead in future if that makes sense, it needs further investigations).

### CODE REVIEW

**Comment by Veetaha** [comment]
Will need to ensure deployment run from master branch only.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    main:
+        runs-on: ubuntu-18.04
+        steps:
+          - name: Checkout this repo
+            uses: actions/checkout@v1
+
+          - name: Install nodejs
+            uses: actions/setup-node@v1.3.0
+            with:
+                node-version: 13.6.0
```
Comment (akrava) [inline .github/workflows/ci.yml]: Maybe is it better to use LTS version of node 12.x? But, probably, it's no sense.

**Comment by akrava** [comment]
What will we push to Heroku: a docker image or single nodejs app?

**Comment by Veetaha** [comment]
@akrava I think this will be a docker image, though this is up to discussion

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Veetaha) [reply .github/workflows/ci.yml]: Nodejs non-lts releases are enough stable on my experience, if we indeed see a critical bug, then we will downgrade to 12.x

**Pull request merged**